src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,254 @@
+        Storage.people.remove(user);
+    }
+
+    @Test
+    void register_passwordLengthIsGreaterThanMinValid_Ok() {
+        user.setPassword("lt84v86ehg");
+        User actual = registrationService.register(user);
+        assertEquals(user, actual,
+                "User with password length more than valid must be registered");
+        Storage.people.remove(user);
+    }
+
+    @Test
+    void register_ageIsNegative_notOk() {
+        user.setAge(-10);
+        assertThrows(RegisterException.class, () ->
+                    registrationService.register(user),
+                "User with negative age can't be registered");
+    }
+
+    @Test
+    void register_ageLessThanValid_notOk() {
+        user.setAge(8);
+        assertThrows(RegisterException.class, () ->
+                    registrationService.register(user),
+                "User with age less than valid can't be registered");
+    }
+
+    @Test
+    void register_ageIsEqualToValid_Ok() {
+        user.setAge(VALID_AGE);
+        User actual = registrationService.register(user);
+        assertEquals(user, actual,
+                "User with age equal to valid can't be registered");
+        Storage.people.remove(user);
+    }
+
+    @Test
+    void register_ageIsGreaterThanValid_Ok() {
+        user.setAge(VALID_AGE);
+        User actual = registrationService.register(user);
+        assertEquals(user, actual,
+                "User with age greater than valid can't be registered");
+        Storage.people.remove(user);
+    }
+
+    @Test
+    void register_userIsNull_notOk() {
+        user = null;
+        assertThrows(RegisterException.class, () ->
+                    registrationService.register(user),
+                "Null user can't be registered");
+    }
+
+    @Test
+    void register_userIdIsNull_notOk() {
+        user.setId(null);
+        assertThrows(RegisterException.class, () ->
+                    registrationService.register(user),
+                "User with null Id can't be registered");
+    }
+
+    @Test
+    void register_userLoginIsNull_notOk() {
+        user.setLogin(null);
+        assertThrows(RegisterException.class, () ->
+                    registrationService.register(user),
+                "User with null Login can't be registered");
+    }
+
+    @Test
+    void register_userPasswordIsNull_notOk() {
+        user.setPassword(null);
+        assertThrows(RegisterException.class, () ->
+                    registrationService.register(user),
+                "User with null Password can't be registered");
+    }
+
+    @Test
+    void register_userAgeIsNull_notOk() {
+        user.setAge(null);
+        assertThrows(RegisterException.class, () ->
+                    registrationService.register(user),
+                "User with null Age can't be registered");
+    }
+
+    @Test
+    void register_userIdAndAgeAreNull_notOk() {
+        user.setId(null);
+        user.setAge(null);

```

Comment:
do not combine diff null value fields. Create one test per one filed for null case
register_ageIsNull_notOk
register_loginIsNull_notOk
register_passwordIsNull_notOk

Reply:
Okay, I see you already have such tests, so you can delete this one and other which test 2 or more null values at once